service/build.gradle

```diff
@@ -202,12 +202,12 @@ dependencies {
   compile group: 'org.flywaydb', name: 'flyway-core', version: '6.3.2'
 
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator', version: versions.springBoot
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: versions.springBoot
```

Comment:
now needed as previously it was transitive dependency of health-spring-boot-starter